src/rust/engine/options/src/arg_splitter.rs

```diff
@@ -26,7 +56,8 @@ pub struct PantsCommand {
     pub goals: Vec<String>,                        // Requested known goals.
     pub unknown_goals: Vec<String>,                // Any requested but unknown goals.
     pub specs: Vec<String>, // What to run against, e.g. targets or files/dirs.
-    pub passthru: Vec<String>, // Any remaining args specified after a -- separator.
+    pub(crate) flags: Vec<Flag>, // Option values set via flag.
+    pub passthru: Option<Vec<String>>, // Any remaining args specified after a -- separator.
```

Comment:
I will do a future pass on visibility, some of these `pub` may be able to be `pub(crate)`. But I wanted to get this bugfix in first.